packages/atoms/connect/ConnectButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+
+import { classNames } from "@calcom/lib";
+
+import { Button } from "../src/components/ui/button";
+import type { AtomsGlobalConfigProps } from "../types";
+import type { ConnectButtonProps } from "../types";
```

Comment:
I think we should consider colocating the type of the button props with the component itself, either in the same file or in the same folder in a different file